yarn.lock

```diff
@@ -8959,6 +8959,21 @@ execa@^5.0.0, execa@^5.1.1:
     signal-exit "^3.0.3"
     strip-final-newline "^2.0.0"
 
+execa@^7.0.0:
```

Comment:
I wanted to use new enough execa for execCommand / native ESM.

Node 14.18 is supported: https://github.com/sindresorhus/execa/blob/main/package.json#L16